### ISSUE TITLE
testDownloadingSameImageWhileInProgressAddsCallbacksWithoutAddingAnotherRequest times out

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-navigation-ios#1314]
[This build](https://www.bitrise.io/build/b417f0eb765fc14e) for #1310 failed with the following test timeout:

```
MapboxNavigationTests.ImageDownloaderTests
  testDownloadingSameImageWhileInProgressAddsCallbacksWithoutAddingAnotherRequest, Asynchronous wait failed: Exceeded timeout of 2 seconds, with unfulfilled expectations: "Second Image Download".
  <unknown>:0
```

Does #1272 contain the fix for this issue? If so, perhaps we could split it out to unblock other work going on in this repository.

/cc @JThramer @akitchen

**Comment by akitchen** [comment]
#1272 was merged, and should help address this.
